Solon_Burleson/blogsII/app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+    has_many :owners, dependent: :destroy
+    has_many :posts, dependent: :destroy
+    has_many :messages, dependent: :destroy
+    EMAIL_REGEX = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]+)\z/i
```

Comment:
Missing: 
1)user has_many blogs through owners 
2)user has_many (blogs)blog_posts through posts
(since owners and posts are the join tables for user and blog, one of the above has_manys need to use a different name(eg blog_posts) along with source) //Refer more relationship tab .... bottom example on the platform